### PR TITLE
preview decks: Explain preview deck view

### PR DESCRIPTION
**Pull request by ywwg**
Did I do this right?

### CODE REVIEW

File: source/chapters/djing_with_mixxx.rst [hunk 1]
```diff
@@ -296,6 +296,14 @@ Deck <interface-preview-deck>` that will only output sound in the
 :ref:`headphones <interface-head-master>` channel. Click the
 |ic_lib_preview_pause| icon to stop the playback.
 
+Mixxx skins also have the option to show a mini Preview Deck which you can
+activate by selecting :menuselection:`View --> Show Preview Deck`.  This will
+show you an overview of the track waveform and you can use the mouse to click
+around the track to preview different parts.  If you want to quickly seek back
+to the beginning of a track, you can right-click the play button in the preview
```
Comment (sblaisot) [inline source/chapters/djing_with_mixxx.rst]: maybe "to the cue point" instead of "to the beginning of a track"

**Comment by rryan** [comment]
Do you want to fix your commit author?

**Comment by rryan** [comment]
buildbot: test this please

**Comment by rryan** [comment]
add to whitelist

**Comment by rryan** [comment]
ping?

**Comment by Pegasus-RPG** [comment]
Looks good, but re-do this PR against manual-2.0.x please.

**Comment by esbrandt** [comment]
Isn't that redundant?
We do have already preview deck blobs in the interface, and the library section
* https://github.com/mixxxdj/manual/blob/manual-2.0.x/source/chapters/user_interface.rst#preview-deck-section
* https://github.com/mixxxdj/manual/blob/manual-2.0.x/source/chapters/library.rst#previewing-tracks

**Comment by Pegasus-RPG** [comment]
I tried to merge the conflicts locally to see what the situation is but my merge tool appears to be auto-merging (incorrectly) but then git deletes the files so I never get the chance.

**Pull request closed without merging**